src/command.ts

```diff
@@ -317,9 +321,10 @@ export class CommandBuilder implements PromiseLike<CommandResult> {
         case "null":
         case "piped":
           return kind;
-        default:
-          const _assertNever: never = kind;
+        default: {
+          const _assertNever: never = kind; //TODO whats this trick
```

Comment:
It's a way to assert that `kind` is a `never` type. If you remove one of the cases above or add another value to `ShellPipeWriterKind` then it should error.